content/en/docs/instrumentation/ruby/manual.md

```diff
@@ -29,70 +81,145 @@ end
 
 ### Creating New Spans
 
-Auto-instrumentation can show the shape of requests to your system, but only you know the really important parts.
-In order to get the full picture of what's happening, you will have to add manual instrumentation and create some custom spans.
-To do this, grab the tracer from the OpenTelemetry API and generate a span:
+To create a [span](/docs/concepts/signals/traces#spans-in-opentelemetry), you’ll
+need a [configured `Tracer`](#acquiring-a-tracer).
+
+Typically when you create a new span, you'll want it to be the active/current
+span. To do that, use `in_span`:
 
 ```ruby
-# ...
+require "opentelemetry/sdk"
 
-def search_by(query)
-  tracer = OpenTelemetry.tracer_provider.tracer('my-tracer')
-  tracer.in_span("search_by") do |span|
-    # ... expensive query
+def do_work()
```

Comment:
```suggestion
def do_work
```

Ruby style nit, we don't typically define methods with parens if they don't take arguments (although some people have strong feelings the opposite way, like all good debates in technology).